pycytominer/operations/transform.py

```diff
@@ -67,58 +73,62 @@ def fit(self, X, y=None):
         self
             With computed weights attribute
         """
-        # Get the mean of the features (columns) and center if specified
-        self.mu = X.mean()
```

Comment:
This should be X.mean(axis=0) or assert X is pd.DataFrame object. If X is numpy array, then X.mean() will be a scalar(global mean)

Reply:
nvm 😅, X.values line below is implicitly assuming X is a dataframe.

Reply:
Right, I do this within `fit`

```
        # Get Numpy representation of the DataFrame
        X = X.values
```

Maybe I should have renamed the variance but keeping it `X` makes it cleaner

Reply:
The new changes have made this outdated so I will resolve